database/database.go

```diff
@@ -486,3 +488,29 @@ func (s *DatabaseService) DeleteExecutionPayloads(idFirst, idLast uint64) error
 	_, err := s.DB.Exec(query, idFirst, idLast)
 	return err
 }
+
+func (s *DatabaseService) SaveValidatorRefund(bidTrace *common.BidTraceV2, signedBlindedBeaconBlock *types.SignedBlindedBeaconBlock) error {
+	_signedBlindedBeaconBlock, err := json.Marshal(signedBlindedBeaconBlock)
+	if err != nil {
+		return err
+	}
+
+	validatorRefundEntry := ValidatorRefundEntry{
+		SignedBlindedBeaconBlock: NewNullString(string(_signedBlindedBeaconBlock)),
+
+		Slot:  bidTrace.Slot,
+		Epoch: bidTrace.Slot / uint64(common.SlotsPerEpoch),
+
+		BuilderPubkey:  bidTrace.BuilderPubkey.String(),
+		ProposerPubkey: bidTrace.ProposerPubkey.String(),
```

Comment:
The `Slot`, `Epoch` and `ProposerPubkey` may be redundant information already available in `SignedBlindedBeaconBlock`. Is the thinking that this may be useful for debugging?

Reply:
I was following the style of the DeliveredPayload table: https://github.com/flashbots/mev-boost-relay/blob/main/database/migrations/001_init_database.go#L82-L105. I think it probably makes the db queries faster and easier to read by directly having these columns, but happy to just leave it as the signed blinded block. I wasn't sure so went with the more verbose option.

Reply:
Ok perfect, makes sense :)